flink-end-to-end-tests/flink-tpcds-test/tpcds-tool/README.md

```diff
@@ -1,19 +1,21 @@
 # TPC-DS tool
 
-This directory contains the necessary scripts/files to run a TPC-DS test.
+This directory contains the necessary scripts/files to run a [TPC-DS]( http://www.tpc.org/tpcds ) benchmark.
 
 ## data_generator.sh scripts
 
-Scripts that to download the TPC-DS data generator and generate test data, the supported OS including GNU/Linux OS and Mac OS.
-
-* parameter 1: generator_dir, the directory to save TPC-DS data generator.
-* parameter 2: scale_factor, the TPC-DS test scale factor, we use the validation scale factor 1 (1G) by default.
-* parameter 3: data_dir, the directory to save TPC-DS test data.
-* parameter 4: retry_scripts_dir, the directory which contains some common scripts for end-to end tests, the retry scripts will be used when download the TPC-DS data generator.  
+Scripts to download the TPC-DS data generator and generate test data, the supported OS including GNU/Linux OS and Mac OS.
+```
+ data_generator.sh <generatorDir> <scaleFactor> <outputDataDir> <commonScriptsDir>
```

Comment:
give some examples?

Reply:
`data_generator.sh` scripts is internally used by `test_tpcds.sh`, and it's only used by `test_tpcds.sh`.
It's still difficult for users to run a TPC-DS bench mark without `test_tpcds.sh` even I give an example.
So, I give an example of `test_tpcds.sh` and also post the use position in` test_tpcds.sh`, HDYT?